public/app/features/plugins/admin/state/selectors.ts

```diff
@@ -19,11 +19,20 @@ const selectInstalled = (filterBy: string) =>
     plugins.filter((plugin) => (filterBy === 'installed' ? plugin.isInstalled : !plugin.isCore))
   );
 
-const findByInstallAndType = (filterBy: string, filterByType: string) =>
+const findByInstallAndType = (filterBy: string, filterByType: PluginTypeFilterOption) =>
   createSelector(selectInstalled(filterBy), (plugins) =>
-    plugins.filter((plugin) => filterByType === 'all' || plugin.type === filterByType)
+    plugins.filter(
+      (plugin) =>
+        filterByType === 'all' ||
+        plugin.type === filterByType ||
+        (filterByType === 'app' && isAppPluginType(plugin.type))
+    )
   );
 
+const isAppPluginType = (type: PluginType | undefined) => {
+  return type === PluginType.renderer || type === PluginType.secretsmanager;
+};
```

Comment:
Although currently this works properly for the use-case above (in `findByInstallAndType()`), shouldn't it also check if the type is `PluginType.app` so it is doing more what it says? (What I am thinking about is if someone would come across this function in the future and tries to reuse it then they can get confused why it is not checking if it is actually an app plugin type)

```javascript
return type === PluginType.app || type === PluginType.renderer || type === PluginType.secretsmanager;
```

Reply:
@leventebalogh ha, you are right, silly oversight by me. Fixed on https://github.com/grafana/grafana/pull/55597/commits/88181b614c86eb4fa6c2c266cbf5b843a6f2e943